src/routes/api/events/+server.js

```diff
@@ -0,0 +1,47 @@
+export async function POST({ request }) {
+	try {
+		const {
+			domain,
+			name,
+			url,
+			referrer,
+			props,
+			apiHost = 'https://plausible.io'
```

Comment:
I'd like to see this design changed slightly: the domain and apiHost can both be pulled directly from the environment variables instead of being passed in from the frontend.